oak/server/oak_manager.h

```diff
@@ -24,7 +26,7 @@ namespace oak {
 
 class OakManager final : public Manager::Service {
  public:
-  OakManager();
+  explicit OakManager(const std::string& enclave_path);
```

Comment:
Worth using `string_view` here? https://abseil.io/tips/1

Reply:
It doesn't seem very useful with only one caller that we control, but sure why not...  :)